types/smooch/index.d.ts

```diff
@@ -180,769 +180,87 @@ declare namespace Smooch {
-        /**
-         * Enables the image upload menu item.
-         * @default true
-         */
-        imageUpload?: boolean;
-        /**
-         * Enables the file upload menu item.
-         * @default true
-         */
-        fileUpload?: boolean;
-        /**
-         * Enables the share location menu item.
-         * @default true
-         */
-        shareLocation?: boolean;
-    }
-
-    interface PrechatCapture {
-        /**
-         * Sets the URL of the avatar to use for the automatic reply to the prechat capture messages.
-         */
-        avatarUrl?: string;
-        /**
-         * Enables the prechat capture experience.
-         * @default false
-         */
-        enabled?: boolean;
-        /**
-         * Automatically links the user's email to the app's Mailgun integration if it exists. If the property fields is defined, the first field of type email will be used.
-         * @default true
-         */
-        enableEmailLinking?: boolean;
-        /**
-         * Overrides the default Prechat Capture fields to define a custom form.
-         */
-        fields?: PrechatCaptureField[];
-    }
-
-    interface PrechatCaptureField {
-        type: 'text' | 'select' | 'email';
-        name: string;
-        label: string;
-        placeholder: string;
-        options?: PrechatCaptureFieldOptions[];
-    }
-
-    interface PrechatCaptureFieldOptions {
-        name: string;
-        label: string;
-    }
-
-    interface User {
-        id: string;
-        externalId: string;
-        signedUpAt: string;
-        hasPaymendInfo: boolean;
-        givenName: string;
-        surname: string;
-        email: string;
-        avatarUrl: string;
-        metadata: unknown;
-        properties: unknown;
-    }
-
-    interface ConversationParticipant {
-        id: string;
-        userId: string;
-        unreadCount: number;
-        lastRead: number;
-    }
-
-    interface Message {
-        role: 'user' | 'business';
-        userId: string;
-        displayName: string;
-        id: string;
-        type: 'text' | 'formResponse' | 'form';
-        received: number;
-        text: string;
-        source: MessageSource;
-    }
-
-    interface SimpleMessage {
-        type: string;
-        text: string;
-    }
-
-    interface MessageSource {
-        type: 'web' | 'slack';
-        id: string;
-        integrationId: string;
-    }
-
-    interface Conversation {
-        id: string;
-        lastUpdatedAt: number;
-        businessLastRead: number;
-        description: string;
-        displayName: string;
-        iconUrl: string;
-        type: 'sdkGroup' | string;
-        participants: ConversationParticipant[];
-        metadata: {};
-        messages: Message[];
-    }
-
-    interface Status {
-        status: 'resolved' | 'pending' | string;
-    }
-
-    interface ConversationData {
-        conversation: Conversation;
-    }
-
+    // tslint:disable:unified-signatures
```

Comment:
Each definition can change independently in the future, therefore its best to keep them separated. 

It also helps with comments and readability.